### PR TITLE
{vis}[intel/2018a] motif v2.3.8

### PR DESCRIPTION
**Pull request by SimonPinches**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/m/motif/motif-2.3.8-intel-2018a.eb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+ It was the standard toolkit for the Common Desktop Environment and thus for Unix."""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['%(name)s-%(version)s.tar.gz']
+checksums = ['859b723666eeac7df018209d66045c9853b50b4218cecadb794e2359619ebce7']
+
+dependencies = [
+    ('X11', '20171023'),
```
Comment (boegel) [inline easybuild/easyconfigs/m/motif/motif-2.3.8-intel-2018a.eb]: @SimonPinches For several of these dependencies we should use more recent versions, and then try to stick to those for the `2018` 'cycle'.

Cfr. #5810 for X11, we should definitely look into updating `libpng` and `freetype` too for use in easyconfigs using a `2018a` toolchain.

Comment (SimonPinches) [reply easybuild/easyconfigs/m/motif/motif-2.3.8-intel-2018a.eb]: I guess this will need to wait until a new X11 version is agreed upon?

Comment (boegel) [reply easybuild/easyconfigs/m/motif/motif-2.3.8-intel-2018a.eb]: #5810 is merged now, so you can update this to `20180131`

File: easybuild/easyconfigs/m/motif/motif-2.3.8-intel-2018a.eb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['%(name)s-%(version)s.tar.gz']
+checksums = ['859b723666eeac7df018209d66045c9853b50b4218cecadb794e2359619ebce7']
+
+dependencies = [
+    ('X11', '20180131'),
+    ('libpng', '1.6.32'),
+    ('xbitmaps', '1.1.1', '', True),
+    ('freetype', '2.8'),
```
Comment (boegel) [inline easybuild/easyconfigs/m/motif/motif-2.3.8-intel-2018a.eb]: please bump to `2.9` to fix conflicts

File: easybuild/easyconfigs/m/motif/motif-2.3.8-intel-2018a.eb [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    ('libjpeg-turbo', '1.5.2'),
```
Comment (boegel) [inline easybuild/easyconfigs/m/motif/motif-2.3.8-intel-2018a.eb]: please bump to `1.5.3` to fix conflicts

File: easybuild/easyconfigs/m/motif/motif-2.3.8-intel-2018a.eb [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['%(name)s-%(version)s.tar.gz']
+checksums = ['859b723666eeac7df018209d66045c9853b50b4218cecadb794e2359619ebce7']
+
+dependencies = [
+    ('X11', '20180131'),
+    ('libpng', '1.6.32'),
```
Comment (boegel) [inline easybuild/easyconfigs/m/motif/motif-2.3.8-intel-2018a.eb]: please bump to `1.6.34` to fix conflicts

File: easybuild/easyconfigs/m/motif/motif-2.3.8-intel-2018a.eb [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    ('xbitmaps', '1.1.1', '', True),
+    ('freetype', '2.8'),
+    ('libjpeg-turbo', '1.5.2'),
+    ('bzip2', '1.0.6'),
+]
+builddependencies = [
+    ('Autotools', '20170619'),
+    ('flex', '2.6.4'),
+    ('Bison', '3.0.4'),
+    ('util-linux', '2.31'),
```
Comment (boegel) [inline easybuild/easyconfigs/m/motif/motif-2.3.8-intel-2018a.eb]: please bump to `2.31.1` (see existing `motif-2.3.8-foss-2018a.eb`)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2518.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/a6a0655f12718f36f651b03943d1d3e2 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
gligar01.gligar.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/0086622c780c0dbf3c1b873fc2e974e9 for a full test report.

**Pull request merged**